problems/sevenBoom.js

```diff
@@ -4,8 +4,18 @@
 * @param {number} n - The number to count up to
 * @returns {number[]} - An array matching the pattern described above
 */
-function sevenBoom() {
-
+function sevenBoom(n) {
+    let i = 1;
+    let arr = [];
+    while (i <= n) {
+      if (i % 7 === 0 && i % 2 !== 0 && i % 1 === 7) {
```

Comment:
We want our conditional to return true if the number is divisible by 7 OR includes the number 7 as a digit. I see that your conditional check's if the number is divisible by 7 AND if it is not an even number AND if it has a remainder of 7, which checks for a lot more than we need it to. Consider how I worded the above statement when rewriting this function.